packages/design-system-react/src/components/button-secondary/ButtonSecondary.tsx

```diff
@@ -42,12 +42,15 @@ export const ButtonSecondary = React.forwardRef<
           // Only apply hover/active styles when interactive
           isInteractive && ['hover:bg-hover', 'active:bg-pressed'],
         ],
-      // Inverse danger styles
+      // Inverse danger styles (matches ButtonPrimary
       isInverse &&
         isDanger && [
-          'bg-transparent border-2 border-error-inverse text-error-inverse',
+          'bg-default text-error-default',
           // Only apply hover/active styles when interactive
-          isInteractive && ['hover:bg-hover', 'active:bg-pressed'],
+          isInteractive && [
+            'hover:bg-default-hover',
+            'active:bg-default-pressed',
+          ],
```

Comment:
Updating classnames to match ButtonPrimary

Reply:
[This note](https://docs.google.com/document/d/19B5Zxfrp53qOYSzQv-N0_QXTULzBwmITjXkD8cqHS8w/edit?tab=t.0) indicates that the background should be transparent

Reply:
This is for when the inverse and danger is true have updated the default hover and pressed states